pydantic/main.py

```diff
@@ -587,10 +587,13 @@ def construct(cls: Type['Model'], _fields_set: Optional['SetStr'] = None, **valu
         m = cls.__new__(cls)
         fields_values: Dict[str, Any] = {}
         for name, field in cls.__fields__.items():
-            if name in values:
+            if field.alt_alias and field.alias in values:
+                fields_values[name] = values[field.alias]
+            elif name in values:
                 fields_values[name] = values[name]
             elif not field.required:
                 fields_values[name] = field.get_default()
+
```

Comment:
Please remove this change. seems unrelated

Reply:
Will do, black did this when I ran it and I don't think it picked up on the local config file for whatever reason, but I didn't make a dedicated python env just for this when I cloned it so it's probably my fault.